projects/fusion-ui/components/input-inline/common/base/input-inline.base.component.ts

```diff
@@ -208,4 +234,18 @@ export abstract class InputInlineBaseComponent implements ControlValueAccessor,
             }
         }
     }
+
+    private isClickInsideByCoordinates(event: MouseEvent): boolean {
+        const parentRect = this.elementRef.nativeElement.getBoundingClientRect();
+        return (
+            parentRect.left <= event.clientX &&
+            parentRect.right >= event.clientX &&
+            parentRect.top <= event.clientY &&
+            parentRect.bottom >= event.clientY
+        );
+    }
+
+    private isType(type: InlineInputType): boolean {
```

Comment:
why ?
why not comapre the enums ?

Reply:
You are right it looks weird, but sometimes (in the storybook) I get it as a string like "InlineInputType.Currency". So just added one more check. Will check it again.